dd4t-api/src/main/java/org/dd4t/contentmodel/Model.java

```diff
@@ -0,0 +1,7 @@
+package org.dd4t.contentmodel;
+
+import java.util.Map;
+
+public interface Model {
```

Comment:
To me it seems this interface is not necessary.. What is the reason to want this?